docs/process/testing.md

```diff
@@ -227,11 +232,22 @@
     - [ ] Merge hint shows status and branches to be merged
     - [ ] `Merge` button only activated if something to merge (includes awaiting conflicts)
       - [ ] If merged, success banner is shown temporarily; user can `X` to remove
-      - [ ] If conflicts, conflict modal shows quality of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
+      - [ ] If conflicts, conflict modal shows quantity of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
         - [ ] If conflict resolved, files marked green, and user can click `Commit merge` button
 	    - [ ] If merged, success banner is shown temporarily; user can `X` to remove
 	    - [ ] Binary files must be resolved in command line before committing merge
         - [ ] Aborting partially resolved commit surfaces "Are you sure?" dialogue; `Cancel` or `Abort merge` buttons
+  - [ ] Rebase Current Branch...
+    - [ ] Use can filter to find existing branches
+    - [ ] User can select branch, other than current one
+    - [ ] Rebase hint shows status and branches to be merged
+    - [ ] `Start rebase` button only activated if something to rebase
+      - [ ] If rebase, success banner is shown temporarily; user can `X` to remove
+      - [ ] If conflicts, conflict modal shows quanity of files, ability to open in [editor], open in command line, `Abort` button, while `Continue rebase` button is deactivated until files are resolved
+        - [ ] If rebase resolved, files marked green, and user can click `Continue rebase` button
+	    - [ ] If rebased, success banner is shown temporarily; user can `X` to remove
+	    - [ ] Binary files must be resolved in command line before committing rebase
```

Comment:
Is this accurate? Don't we have support for choosing binary files in desktop now?

Reply:
You are correct. I had copy/pasted the merge conflict steps for rebasing. Fixing it.